public/css/main.css

```diff
@@ -88,6 +88,11 @@ input[type="checkbox"]+label:before {
 	cursor: pointer;
 }
 
+input[type="radio"]:focus+label:before,
+input[type="checkbox"]:focus+label:before {
+	box-shadow: 0 0 0 3px #dcb9d2;
```

Comment:
Task card says "blue" but the pinky colour #dcb9d2 works; amend card to make this clear?

Reply:
True!! Thanks.